movies/src/main/java/se/mulander/cosmos/movies/business/Movies.java

```diff
@@ -81,5 +82,20 @@ public Response setMovieObjectStatus(@ApiParam(value = "The id to search for") @
                                            String status) {
         return se.mulander.cosmos.movies.impl.Movies.setMovieStatus(id, status);
     }
+
+    @GET
+    @Path("/search/{query}")
```

Comment:
Den här Pathen är konstig då den inte tar in en path param.
Borde väl ändras till "/search" och låta query vara kvar som queryparam